source/npm/util.js

```diff
@@ -109,3 +111,57 @@ exports.checkIgnoreStrategy = ({files}) => {
 		`);
 	}
 };
+
+// New files added part of .npmignore or not part of files-attribute in "package.json"
+exports.checkNewFiles = async (newFiles, filesFromFileAttribute) => {
```

Comment:
I'd rename this method to `getPublishedFiles`, as this method returns all files which will be published as part of the package.

Reply:
I called it `getNewAndUnpublishedFiles`. I think it describes the intention behind this function best.